### PR TITLE
test(FE): e2e test for plugin page

### PR DESCRIPTION
**Pull request by idbeta**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] E2E
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
close https://github.com/apache/apisix-dashboard/issues/1156

### CODE REVIEW

File: web/cypress/integration/plugin/create-edit-delete-plugin.spec.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  beforeEach(() => {
+    // init login
+    cy.login();
+  });
+
+  it('should enable two plugins', () => {
+    //  go to route plugin list page
+    cy.visit('/');
+    cy.contains('Plugin').click();
```
Comment (LiteSun) [inline web/cypress/integration/plugin/create-edit-delete-plugin.spec.js:30]: can we remove `cy.wait()` and write like this?

```suggestion
    cy.contains('Plugin').click({timeout: xxx});
```

File: web/cypress/integration/plugin/create-edit-delete-plugin.spec.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    cy.wait(sleepTime * 5);
+    cy.contains('Create').click();
+    cy.wait(sleepTime * 5);
```
Comment (moonming) [inline web/cypress/integration/plugin/create-edit-delete-plugin.spec.js:33]: as discuss before, we don't need cy.wait

**Review by moonming — CHANGES_REQUESTED** [review]

File: web/cypress/integration/plugin/create-edit-delete-plugin.spec.js (hunk 1 above)
Comment (idbeta) [reply web/cypress/integration/plugin/create-edit-delete-plugin.spec.js]: fixed, thanks for reminding.

File: web/cypress/integration/plugin/create-edit-delete-plugin.spec.js (hunk 2 above)
Comment (idbeta) [reply web/cypress/integration/plugin/create-edit-delete-plugin.spec.js]: fixed, thanks for reminding.

File: web/cypress/integration/plugin/create-edit-delete-plugin.spec.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+context('e2e test for plugin page', () => {
+  const sleepTime = 50000;
+
+  beforeEach(() => {
+    // init login
+    cy.login();
+  });
+
+  it('should enable two plugins', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/plugin/create-edit-delete-plugin.spec.js:27]: https://github.com/apache/apisix-dashboard/pull/1308/files#diff-f7a862c3853a71e325c09d7d25c05ccd96ba223ad611a71302d3547e8163275cR45-R85

Please refer to this PR.

File: web/cypress/integration/plugin/create-edit-delete-plugin.spec.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    //  go to route plugin list page
+    cy.visit('/');
+    cy.contains('Plugin').click({ force: true, timeout, });
+    cy.contains('Create').click({ force: true, timeout, });
+
+    // enable auth plugin
+    cy.contains('.ant-card', 'key-auth').within(() => {
```
Comment (moonming) [inline web/cypress/integration/plugin/create-edit-delete-plugin.spec.js:34]: Can the auth plugin enabled without consumers?

Comment (idbeta) [reply web/cypress/integration/plugin/create-edit-delete-plugin.spec.js:34]: Now the dashboard allows this operation.

Comment (juzhiyuan) [reply web/cypress/integration/plugin/create-edit-delete-plugin.spec.js:34]: Global Auth Plugins don't need extra configuration no matter whether the related consumer exits.

**Comment by idbeta** [comment]
After removing the `cy.wait()` of this use case, CI failed , so I add it back now,  please analyze if you have time, @LiteSun. I will raise an issue later.

**Pull request closed without merging**